front/src/applications/rollingStockEditor/components/RollingStockEditorForm.tsx

```diff
@@ -159,8 +177,8 @@ const RollingStockEditorForm = ({
       type: rollingStockData?.metadata.type || '',
       unit: rollingStockData?.metadata.unit || '',
       length: rollingStockData?.length || 0,
-      mass: rollingStockData?.mass || 0,
-      max_speed: rollingStockData?.max_speed || 0,
+      mass: rollingStockData ? rollingStockData.mass / 1000 : 0,
```

Comment:
Can you specify the unit if you're making a conversion ?

Reply:
Comments added